lib/facilities/vba_facility.rb

```diff
@@ -0,0 +1,16 @@
+# frozen_string_literal: true
+
+require 'facilities/shared_client'
+
+module Facilities
+  class VBAFacility < BaseFacility
+    FACILITY_TYPE = 'va_benefits_facility'
+    default_scope { where(facility_type: FACILITY_TYPE) }
+
+    class << self
+      def pull_source_data
+        SharedClient.new.get_all_vba.map(&method(:new))
+      end
+    end
+  end
```

Comment:
I think there is an opportunity to dynamically determine the SharedClient methods and DRY this up a bit. Perhaps moving some of these methods on to BaseFacility.

Additionally, consider using type instead of default scope.

```
  set_inheritance_column :facility_type

  FACILITY_MAPPINGS = {
...
  }

  class << self
    def find_sti_class(type_name)
      FACILITY_MAPPINGS[type_name.to_i] or super
    end

    def sti_name
     FACILITY_MAPPINGS.invert[self]
    end
  end
```